cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -137,15 +142,15 @@ Future<Void> reconcile(ReconciliationState reconcileState)  {
         reconcileState.initialStatus()
                 // Preparation steps => prepare cluster descriptions, handle CA creation or changes
                 .compose(state -> state.reconcileCas(this::dateSupplier))
-                .compose(state -> state.versionChange())
+                .compose(ReconciliationState::versionChange)
```

Comment:
I find it a bit weird when each line is using a different pattern. So I was leaving these intentionally TBH.

Reply:
Fair enough, will revert it.